trikControl/src/i2cBusCommunicator.cpp

```diff
@@ -0,0 +1,86 @@
+/* Copyright 2018 Aleksey Fefelov and CyberTech Labs Ltd.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License. */
+
+#include "i2cBusCommunicator.h"
+
+using namespace trikControl;
+using namespace trikHal;
+
+#include <trikKernel/configurer.h>
+#include <trikHal/mspI2cInterface.h>
+#include <QsLog.h>
+
+
+I2cBusCommunicator::I2cBusCommunicator(trikHal::MspI2cInterface &i2c, uint8_t bus, uint8_t deviceId)
```

Comment:
Why MspI2cInterface is in use here? MSP is just an I2C device.

Reply:
trikMspI2c is hardware implementation of work with i2c